src/core/Akka/Actor/ActorPath.cs

```diff
@@ -298,10 +297,14 @@ private static bool TryParseAddress(string path, out Address address, out Uri ur
             //This code corresponds to AddressFromURIString.unapply
             uri = null;
             address = null;
-
-            if (!Uri.TryCreate(path, UriKind.Absolute, out uri))
+            try
```

Comment:
Anything that helps fix clustering should just go out.
On Wed, Mar 23, 2016 at 7:56 AM Joshua Benjamin <joshuabnjmn@gmail.com>
wrote:

> Doesn't matter. Cluster code past 1.0.3 is broken anyways
>
> On Wed, Mar 23, 2016 at 7:26 AM Jeff Cyr <notifications@github.com> wrote:
>
>> In src/core/Akka/Actor/ActorPath.cs
>> <https://github.com/akkadotnet/akka.net/pull/1817#discussion_r57167412>:
>>
>> > @@ -298,10 +297,14 @@ private static bool TryParseAddress(string path, out Address address, out Uri ur
>> >              //This code corresponds to AddressFromURIString.unapply
>> >              uri = null;
>> >              address = null;
>> > -
>> > -            if (!Uri.TryCreate(path, UriKind.Absolute, out uri))
>> > +            try
>>
>> Why not TryCreate?
>>
>> —
>> You are receiving this because you are subscribed to this thread.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/akkadotnet/akka.net/pull/1817/files/bf9d9f3e6c8cb73876b2bea8c8b7788c84a51d1e#r57167412>
>>
>